dubbo-common/src/main/java/org/apache/dubbo/config/AbstractInterfaceConfig.java

```diff
@@ -420,7 +420,7 @@ protected boolean verifyMethodConfig(
     }
 
     private ArgumentConfig getArgumentByIndex(MethodConfig methodConfig, int argIndex) {
-        if (methodConfig.getArguments() != null && methodConfig.getArguments().size() > 0) {
+        if (methodConfig.getArguments() != null && !methodConfig.getArguments().isEmpty()) {
```

Comment:
**Security Issue**: Using Collection's size() to check for emptiness can be less efficient than using isEmpty(). <br> **Fix**: Use isEmpty() to check for collection's emptiness for better performance. <br> **Code Suggestion**: 
 ```
 -        if (methodConfig.getArguments() != null && methodConfig.getArguments().size() > 0) {
 +        if (methodConfig.getArguments() != null && !methodConfig.getArguments().isEmpty()) {
 ```